terraform/development/main.tf

```diff
@@ -49,6 +56,10 @@ module "elasticsearch_db_development" {
   ebs_volume_size  = "10"
   region           = data.aws_region.current.name
   account_id       = data.aws_caller_identity.current.account_id
+
+  auto_tune_options {
```

Comment:
I'm not sure this will do anything on it's own. Want to pair on this and we can get it in to the common module?

Reply:
Also you've a typo in 'desired'